### DISCUSSION TITLE
Airflow scheduling task has high latency

### DISCUSSION THREAD

**Discussion by hrbust-liu** [apache/airflow#43105] [Q&A]
We found that the time difference between the two concurrent tasks of the same Dag is more than half an hour.
After checking the Airflow Scheduler log, we found that 1 Airflow reported an error "MySQL server has gone away"
So, will too many dags cause abnormal connection to MySQL? I have 3,000 dags, and about 30 of them are running at the same time. My airflow always reports an error "MySQL server has gone away". Whether I use local docker mysql or rds on the cloud, I can't solve this problem

**Comment by potiuk** [comment]
You need to see at your MySQL server logs to see what the problem is. Your server had gone away and it writes in their logs "why" - without it is impossible to help you to solve your problem. It could be memory issue (your server has too little memory) or a number of other issues - but only you looking at your logs and understanding why your MySQL server crashed can diagnose it.
